src/app/main/main.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-main',
+  templateUrl: './main.component.html',
+  styleUrls: ['./main.component.css']
+})
+
+export class MainComponent implements OnInit {
```

Comment:
This component isn't actually required. You can have the html written in this file directly in the `app.component.html` as this only contains `router-outlet` and any css styles

Reply:
`main.component` removed and `router-outlet` code added to `app.component.html`